src/Divi/CustomFields/MultiSelect/index.js

```diff
@@ -0,0 +1,42 @@
+import { Component } from 'react';
+// eslint-disable-next-line no-unused-vars
+import MultiSelect from '@khanacademy/react-multi-select';
+
+import './style.css';
+
+class SimpleInput extends Component {
```

Comment:
We should probably updated the class name to be more descriptive (I assuming that SimpleInput comes from the docs),

Reply:
Good catch! Yeah, it is from the example in the docs.
Resolved 33d859d